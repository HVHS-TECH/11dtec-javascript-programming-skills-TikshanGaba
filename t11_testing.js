/****************************
 Name of task: Testing
 Name: Tikshan
 Date: Term two week three 2025
 Purpose:  Learning javascript
*****************************/

//Variables
let userName;
let userAge;
let userMoney;
/****************************
Main Code
****************************/
// start()
// Runs the program
// Called by the start button
userName ();
userAge ();
userMoney ();
/**************************** 
Functions
****************************/
function start()
 {
    userName = prompt("Oh hi, Maybe I forgot you.. What was your name again ?");
    userAge = prompt("Nice name " + userName + ", how old are you ?");
    userMoney = prompt("Oh you're " + userAge + ", how much pocket money do you get ?");
    alert("So, you're "+ userName + ", and you're "+ userAge + " years old and you get " + userMoney + " dollars as pocket money. Nice")
 }
