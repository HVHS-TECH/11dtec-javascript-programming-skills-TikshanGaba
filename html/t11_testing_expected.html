<!DOCTYPE html>
<html lang="en">
<head>
  <title>T11</title>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <!-- stylesheets *******************************************-->
  <link rel="stylesheet" href="../css/style.css">
</head>

<!--*************************************************************
  BODY
**************************************************************-->
<body>
  <h1>T11 - Testing expected cases</h1>
  <script src="../t11_testing.js"></script>
   <div class="w3-container">
    <h1>????????</h1>
    <button class="w3-btn w3-????? w3-xxlarge" type="button"
    onclick="start ();">start</button>
  </div>
</body>
</html>
<!--*************************************************************
  END OF T11_TESTING_EXPECTED.HTML
**************************************************************-->